docs/overview/roadmap.md

```diff
@@ -61,92 +61,57 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
-  [future work section](#future-work) for further details). Provide each entity
-  within the software catalog with a recognized owner.
-- **Catalog performance improvements through improved caching:** Fix the
-  performance gaps in the catalog processor, which currently doesn’t have a
-  strong caching mechanism. The current version often requires fetching a
-  relevant amount of data, especially at scale.
+### Backstage 1.0 (and following versions)
+
+During the first quarter of 2022 it is planned to be finalised and released the
+version 1.0 of the Backstage platform (defined by the Core,
+[Catalog](https://backstage.io/docs/features/software-catalog/software-catalog-overview),
+[Scaffolder](https://backstage.io/docs/features/software-templates/software-templates-index)
+and [TechDocs](https://backstage.io/docs/features/techdocs/techdocs-overview)).
+Included as part of this milestone:
+
+- The cadence of minor/weekly/daily releases to provide clarity on the frequency
+  and expectations for future versions of the platform and its defining modules.
+- The support model to set the expectations from the adopters in their
+  respective use cases.
+
+### Backstage Security Audit
+
+This initiative is the first of a broader Security Strategy for Backstage. The
+purpose of the Security Audit is to involve third party companies in auditing
+the platform and highlighting potential vulnerabilities (if there are any). The
+benefit for the adopters is clear: we want Backstage to be as much secure as
+possible and we want to make it reliable through a specific initiative. This
+initiative in particular is done together (and with the support of) the
+[Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/).
+
+### Moving to Incubation in CNCF
+
+The progress of the request can be seen
+[here](https://github.com/cncf/toc/pull/717).
 
 ## Future work
 
 The following feature list doesn’t represent a commitment to develop and the
 list order doesn’t reflect any priority or importance. But these features are on
 the maintainers’ radar, with clear interest expressed by the community.
 
+- **Backend Services:** To better scale and maintain the Backstage instances, a
+  backend layer of services is planned to be introduced as part of the software
+  architecture. This layer of backend services will help in decoupling the
+  various modules (Catalog and Scaffolder just to quote some) from the front-end
+  experience.
+- **Security Plan (and Strategy):** The purpose of the Security Strategy is to
+  move another step ahead along the path of the maturity of the platform,
+  setting the expectations of any adopters from a security point.
+- **Search GA:**.
+- **[GraphQL](https://graphql.org/) support:** Introduce the ability to query
+  Backstage backend services with a standard query language for APIs.
+- **Telemetry:** To efficiently collect, store and analyse system and
+  application log data so that Backstage can be monitored and improved.
```

Comment:
Hm the goal isn't really to collect/store/analyse things, is it? Isn't it to _generate_ logging and metrics in such a way that adopters can get insights?

Reply:
Reviewed the description. Thanks.